PWGHF/HFC/Tasks/taskCorrelationDsHadrons.cxx

```diff
@@ -86,6 +86,9 @@ struct HfTaskCorrelationDsHadrons {
   Configurable<std::string> fdEffCcdbPath{"fdEffCcdbPath", "", "CCDB path for trigger efficiency"};
   Configurable<int64_t> timestampCcdb{"timestampCcdb", -1, "timestamp of the efficiency files used to query in CCDB"};
 
+  const int comingFromCharm = 1;
+  const int comingFromBeauty = 2;
```

Comment:
As far as I can see, the content of the columns is actually a value from `RecoDecay::OriginType`, so you are still hard-coding them.